src/charm.py

```diff
@@ -93,6 +96,18 @@ def __init__(self, *args):
             ports=[(f"{self.app.name}", self._port)],
         )
 
+        # observability endpoints
```

Comment:
I think the correct term here is integrations (formerly: relations)
```suggestion
        # Observability integrations
```